packages/core/src/App/Components/Elements/Modals/RedirectNotice/redirect-notice-modal.jsx

```diff
@@ -32,10 +33,14 @@ const RedirectNoticeModal = ({ is_logged_in, is_eu }) => {
     };
 
     const onClickExternalLink = e => {
-        if (isThirdPartyLink(e.target.href) && is_logged_in && is_eu) {
-            setExternalLink(e.target.href);
-            e.preventDefault();
-            setDialogStatus(true);
+        const link_element = e.target.tagName === 'A' ? e.target : e.target.closest('a');
```

Comment:
(Question) Is it possible to use `e.currentTarget`?

Reply:
`e.currentTarget` gives you the `document`. I used `closest` to get the first parent that has the actual href link we're looking for